From 55d4c4cc9455f348b5bc455675004a0e035ea31d Mon Sep 17 00:00:00 2001
From: Owen W. Taylor <otaylor@fishsoup.net>
Date: Tue, 17 Nov 2009 17:24:53 -0500
Subject: [PATCH] Allow applications to raise windows when raise_on_click is off

Whether Metacity honors a raise request from an application should
not be affected by the raise_on_click setting; remove a check that
seems to have been added in error.

https://bugzilla.gnome.org/show_bug.cgi?id=445447
---
 src/core/window.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/src/core/window.c b/src/core/window.c
index b481ccf..db59de1 100644
--- a/src/core/window.c
+++ b/src/core/window.c
@@ -4716,8 +4716,7 @@ meta_window_configure_request (MetaWindow *window,
     {
       MetaWindow *active_window;
       active_window = window->display->expected_focus_window;
-      if (meta_prefs_get_disable_workarounds () ||
-          !meta_prefs_get_raise_on_click ())
+      if (meta_prefs_get_disable_workarounds ())
         {
           meta_topic (META_DEBUG_STACK,
                       "%s sent an xconfigure stacking request; this is "
-- 
1.6.5.2

